.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.11"
-  - "0.12"
-
+  - "9.11.1"
```

Comment:
i think 9 is no longer maintained; 10 is the current LTS https://nodejs.org/en/about/releases/

Reply:
Thanks upgraded to `10.15.3`, was using outdated template :)